src/deploy/deploy_libraries.ts

```diff
@@ -33,6 +33,13 @@ const deploy: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
         log: true,
         deterministicDeployment: true,
     });
+
+    await deploy("SafeToL2Setup", {
```

Comment:
Shouldn't we also add the `SafeToL2Migration` and `SafeMigration` contract deployments?

Reply:
They are already added in `deploy_migrations.ts`

Reply:
Ah, missed it 👍🏾 So, just have to add those to the `CHANGELOG`.

Reply:
`SafeToL2Migration` is not a new contract and `SafeMigration` address is dependent on constructor args so we can't predict the address beforehand.

Reply:
> SafeMigration needs constructor args.

But we know this at deployment time right? This is the `Safe` version that we just deployed. You may need to setup deployments correctly in the scripts however.

Reply:
> SafeToL2Migration is not a new contract

It is new to 1.4.1 AFAIU